docs/storage_snippets.py

```diff
@@ -138,7 +139,8 @@ def delete_blob(client, to_delete):
     from google.cloud.exceptions import NotFound
     client = storage.Client()
     bucket = client.get_bucket('my-bucket')
-    assert isinstance(bucket.list_blobs(), list)
+    blobs = list(bucket.list_blobs())
+    assert isinstance(blobs, list)
```

Comment:
FWIW, bucket names are in a global namespace:  we should be prefixing them with something like `gcp-storage-` and then a timestamp / random suffix.